### PR TITLE
Feature arc42 03context

### PR DESCRIPTION
**Pull request by sarav22**

### CODE REVIEW

**Comment by UO258425** [comment]
LGTM

File: src/docs/03_system_scope_and_context.adoc [hunk 1]
```diff
@@ -23,44 +23,63 @@ Various options:
...
-The title of the table is the name of your system, the three columns contain the name of the communication partner, the inputs, and the outputs.
-****
 
-**<Diagram or Table>**
+.Viade
+|=========================================================
+| Communication partner |Inputs |Outputs
+
+| User
+| Upload routes. Attaches photos, videos or information to them
```
Comment (luispc1998) [inline src/docs/03_system_scope_and_context.adoc]: And he shares that information with other users.

File: src/docs/03_system_scope_and_context.adoc [hunk 2]
```diff
@@ -23,44 +23,63 @@ Various options:
...
+| Reads routes from its own pod
```
Comment (luispc1998) [inline src/docs/03_system_scope_and_context.adoc]: The user can see the routes in a map.

Comment (UO258425) [reply src/docs/03_system_scope_and_context.adoc]: Thats not an input, thats what the user does with the input

File: src/docs/03_system_scope_and_context.adoc [hunk 3]
```diff
@@ -23,44 +23,63 @@ Various options:
...
-.Contents
-Specification of *all* communication partners (users, IT-systems, ...) with explanations of domain specific inputs and outputs or interfaces.
-Optionally you can add domain specific formats or communication protocols.
 
-.Motivation
-All stakeholders should understand which data are exchanged with the environment of the system.
 
-.Form
-All kinds of diagrams that show the system as a black box and specify the domain interfaces to communication partners.
+image::SystemScopeDiagram.png[System scope diagram]
```
Comment (Angelixus) [inline src/docs/03_system_scope_and_context.adoc]: Image is not shown, there is no image on the images file.

File: src/docs/03_system_scope_and_context.adoc [hunk 4]
```diff
@@ -23,44 +23,63 @@ Various options:
...
 
-**<optionally: Explanation of external domain interfaces>**
 
 === Technical Context
 
-[role="arc42help"]
-****
-.Contents
-Technical interfaces (channels and transmission media) linking your system to its environment. In addition a mapping of domain specific input/output to the channels, i.e. an explanation with I/O uses which channel.
+image::technicalcontext.jpg[Technical context diagram]
```
Comment (Angelixus) [inline src/docs/03_system_scope_and_context.adoc]: Same as on the first image, it is not shown, there is no image on the images file.

Comment (UO258425) [reply src/docs/03_system_scope_and_context.adoc]: Check the commits, two images are uploaded in the corresponding folder

**Pull request merged**